src/__mocks__/eslint.js

```diff
@@ -35,6 +35,7 @@ function mockGetConfigForFile(filePath) {
   return {
     '/mock/default-config': {
       rules: {
+        semi: [2, 'never'],
```

Comment:
Before this PR, this was actually inherited via the configuration from this project.